src/partials/about.html

```diff
@@ -0,0 +1,19 @@
+<section class="about">
+  <div class="about__head">
```

Comment:
Lepiej abyś nazwał to "about__container" lub "about__item" bo po tem będziesz dodawał do nich layout tego fragmentu